engine/src/main/java/org/terasology/rendering/dag/stateChanges/EnableBlending.java

```diff
@@ -19,20 +19,18 @@
 import org.terasology.rendering.dag.StateChange;
 
 /**
- * Enables blending in the OpenGL pipeline.
- * Can be used to render transparent objects or to make a composite of different images.
+ * Enables OpenGL blending.
+ * Blending can be used to render transparent objects on top of those already rendered in a buffer or to make a composite of different images.
  *
- * Note that SetBlendFunction can be used to set the source and destination factors used by the blending process.
+ * See also StateChange implementation SetBlendFunction to set the source and destination factors used by the blending process.
  *
- * OpenGL Default: DisableBlending
- * Terasology Default: DisableBlending
- *
- * Type: Set
- * Corresponding Reset: DisableBlending
+ * By default both OpenGL and Terasology keep blending disabled as it is a relatively expensive operation.
  */
 public final class EnableBlending extends EnableStateParameter {
     private static StateChange defaultInstance = new DisableBlending();
 
+    // TODO: Do we really need to add such a description for every single StateChange?
+    // TODO: IMHO, we can just add the common parts to StateChange interface, and add class-specific info as needed.
```

Comment:
This kind of comment, sort-of directed to the reviewer, can be added as a comment on github rather than in code. I often add comments of my own shortly after I publish a PR, to point the attention of the reviewers to specific parts of the code, to illustrate concerns I might have and to make sure I get additional opinions about specific issues.

That been said, you make a good point here. Make a separate commit to show me what you mean in practice: keep it small, don't invest too much time on it just yet, i.e. modify the StateChange interface javadoc and the javadoc for _just_ this EnableBlending class. Only if we both like the changes we can then roll out the same javadoc format to other classes.